src/MediaHandler.d.ts

```diff
@@ -239,7 +239,7 @@ export declare interface MediaHandlerOptions {
   /**
    * Authentication header for fetching sources.
    */
-  auth?: string,
+  auth?: string|(() => string),
```

Comment:
I guess the function does not need to be async, right?

Reply:
no